usr/lib/libxt_luaxtabletest.c

```diff
@@ -0,0 +1,63 @@
+/*
+* SPDX-FileCopyrightText: (c) 2024 Mohammad Shehar Yaar Tausif <sheharyaar48@gmail.com>
+* SPDX-License-Identifier: MIT OR GPL-2.0-only
+*/
+
+#include <xtables.h>
+#include <linux/netfilter.h>
+
+#include "luaxtable.h"
+
+static void ipaddr_mt_help(void)
+{
+}
+
+static void ipaddr_mt_init(struct xt_entry_match *match)
+{
+}
+
+static int ipaddr_mt_parse(int c, char **argv, int invert,
+    unsigned int *flags, const void *entry, struct xt_entry_match **match)
+{
+	return true;
+}
+
+static void ipaddr_mt_check(unsigned int flags)
+{
+}
+
+static void ipaddr_mt_print(const void *entry,
+    const struct xt_entry_match *match, int numeric)
+{
+}
+
+static void ipaddr_mt_save(const void *entry,
+    const struct xt_entry_match *match)
+{
+}
+
+static struct xtables_match ipaddr_mt_reg = {
+	.version       = XTABLES_VERSION,
+	.name          = "luaxtabletest",
```

Comment:
if it's tied up to your example, this should live just besides it on `examples/`.. similarly to bpf program of `examples/filter`